tests/phpunit/CRM/Contact/BAO/QueryTest.php

```diff
@@ -44,12 +49,89 @@ public function tearDown() {
    * @param $full
    */
   public function testSearch($fv, $count, $ids, $full) {
-    $op = new PHPUnit_Extensions_Database_Operation_Insert();
-    $op->execute($this->_dbconn,
-      $this->createFlatXMLDataSet(
-        dirname(__FILE__) . '/queryDataset.xml'
```

Comment:
if we're no longer using this file, should we remove it?

Reply:
Yep - I was gonna look at those at the end